### DISCUSSION TITLE
如何在Vue3中创建menu

### DISCUSSION THREAD

**Discussion by WhiteZbaize** [Milkdown/milkdown#993] [Q&A]
Milkdown v7已经弃用了menu，但是我的项目中特别需要menu组件，不知道menu组件是被其他组件替代了还是需要自己去实现menu组件，在网上翻了好长时间都没有找到解决方案，希望有人能帮我解决这个疑问！！

**Comment by fishel-feng** [comment]
menu 的本质就是点击按钮调用对应的命令，参考官方 playground 的做法
https://github.com/Milkdown/website/blob/main/src/components/playground-editor/index.tsx
